src/scene/shader-lib/programs/lit-shader.js

```diff
@@ -870,7 +870,7 @@ class LitShader {
                 func.append(chunks.shadowPCSSPS);
             }
 
-            if (!(device.webgl2 || device.extStandardDerivatives || device.isWebGPU)) {
+            if (!(device.isWebGl2 || device.isWebGPU || device.extStandardDerivatives)) {
```

Comment:
Seems that `isWebGl2` doesn't capitalize the `L` but `isWebGPU` capitalizes the `PU`. Kinda inconsistent.

Reply:
true, thanks, changing to all caps, as in the PR description